src/servers/rolemanager.ts

```diff
@@ -14,6 +14,7 @@ import { serviceSpecificationFromName } from "../jdom/spec"
 export interface RoleBinding {
     role: string
     serviceShortId: string
+    client: boolean
```

Comment:
The role manager as it is is still compatible with the service specification, so we can use it as a simulator. What's the goal of this change?

Reply:
Roles are, by default, for programming against (client = true), no change in behavior. When we are doing server blocks, we need roles that we are implement.

Reply:
You are supposed to be on vacation...

Reply:
actually I figured out a way that doesn't need this change to role blocks